Jenkinsfile

```diff
@@ -0,0 +1,107 @@
+pipeline {
+  agent {
+    node {
+      label 'project:any'
+    }
+  }
+  parameters {
+    choice(choices: ['snapshots', 'releases'], description: 'type of build', name: 'BUILD_TYPE')
+    string(defaultValue: 'nldi', description: 'docker image path in artifactory', name: 'ARTIFACTORY_PATH', trim: false)
+    string(defaultValue: 'wma/docker/nldi', description: 'docker image path in gitlab', name: 'GITLAB_PATH', trim: false)
+    string(defaultValue: '579777464052.dkr.ecr.us-west-2.amazonaws.com', description: 'Elastic Container Registry Root', name: 'ECR_PATH', trim: false)
```

Comment:
I'm somewhat leaning towards asking that the default be removed so it can be handled the same way we're doing the Actifactory and GitLab hosts. That is, non-public URLs are not disclosed.